stock_inventory_revaluation_adjustment/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 - Open Source Integrators
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Stock Inventory Revaluation",
+    "summary": "Adds a workflow to adjusting costs on products",
+    "version": "14.0.1.0.0",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "category": "Stock",
+    "license": "AGPL-3",
+    "depends": ["stock_account", "mrp"],
```

Comment:
```suggestion
    "depends": ["stock_account"],
```

Reply:
I agree this module should not be depending on MRP here. 

There are more changes necessary though:

1. remove bom_count on product.template
2. remove bom_ids